en/airframes/airframe_reference.md

```diff
@@ -1052,3 +1052,20 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### VTOL Octoplane
```

Comment:
Please remove all the airframe_reference changes in all languages - this stuff is autgenerated from source so we just need the image.

More generally, and FYI only, we never edit the translations. That stuff is auto exported from our translation tool crowdin.

Reply:
All done
Did not know this, thanks for explaining :)

Reply:
Thanks very much. It is documented, but very easily missed given how many other things you need to think about : https://dev.px4.io/master/en/airframes/adding_a_new_frame.html#adding-a-new-airframe-group